packages/code-studio/src/index.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import 'fira';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
I don't think disabling this line is necessary?
```suggestion
```

Reply:
I think it's required b/c otherwise the scss from components needs to be built for the import to resolve. I can't do the paths imports that I did for the few imports from `iris-grid/dist` because it's a css file

This is the only place that imports css from another package and it should remain that way